packages/flutter_tools/lib/src/web/module_metadata.dart

```diff
@@ -0,0 +1,189 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
This file is just a duplicate copy, no modifications right? Maybe we should leave some kind of link in the comment to the original source of truth?

Reply:
Mostly. It required a couple tweaks to make type check-related lints happy within this repo.

Done.